Python/Product/Analysis/AnalysisNetStandard.csproj

```diff
@@ -34,6 +29,9 @@
     <Content Include="typing-stub.pyi">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="Typeshed\**\*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
VS Code extension includes typeshed as submodule to the extension rather than fixed version with the LS. Packaging it with LS may yield two typesheds installed.

Reply:
Brett made an issue on the PVSC repo to remove the submodule and rely on this copy. (Importantly, I think we may need to modify typeshed's `sys.pyi` to make it work properly - it currently imports itself, which may be the cause of some typeshed issues.)